server/build-plugins.js

```diff
@@ -13,6 +13,8 @@ const logger = require('./logger');
 const config = new Config('bpanel');
 config.load({ env: true, argv: true, arg: true });
 const pluginsConfig = resolve(config.prefix, 'config.js');
+const modulesDirectory = resolve(__dirname, '../node_modules');
+const homePrefix = process.env.BPANEL_PREFIX;
```

Comment:
this gets passed by the webpack process run by server/index.js.

Reply:
Maybe add a comment around that?